controllers/activemqartemis_controller_deploy_operator_test.go

```diff
@@ -0,0 +1,153 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// +kubebuilder:docs-gen:collapse=Apache License
+
+/*
+As usual, we start with the necessary imports. We also define some utility variables.
+*/
+package controllers
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"os"
+
+	"github.com/artemiscloud/activemq-artemis-operator/pkg/resources/configmaps"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+var _ = Describe("artemis controller", Label("doh"), func() {
```

Comment:
I see `doh` label works but  `test-mk-do` has the filter `do`
```suggestion
var _ = Describe("artemis controller", Label("do"), func() {
```

Reply:
ah, forgot that. Thanks!